integration-tests/slack/slack-notify-channels.json

```diff
@@ -0,0 +1,172 @@
+{
+  "pass": [
+    {
+      "name": "dsde-qa",
+      "id": "C53JYBV9A"
+    }
+  ],
+  "fail": [
+    {
+      "name": "dsde-qa",
+      "id": "C53JYBV9A",
+      "tests": [
```

Comment:
I'm assuming we want to always notify `#dsde-qa` for any test failure? If so, it would be nice if we didn't have to list every test here. Could the `tests` key be made optional and if it is not specified, then that Slack channel is notified for any test failure?

Reply:
Yes, idea is notifiy `#dsde-qa` channel for failed tests on `dev` branch on CircleCI.
It's a good idea. I have a prototype working but it doesn't work for that change. I'll take out tests in `#dsde-qa` channel for now and play around with changes in my prototype later.